cli/cli.go

```diff
@@ -35,6 +36,14 @@ type Initializer interface {
 
 func (f *Flags) Init(all ...Initializer) (context.Context, context.CancelFunc, error) {
 	if f.showVersion {
+		if Version == "" {
+			Version = "unknown"
+			if info, ok := debug.ReadBuildInfo(); ok {
+				// This will be "(devel)" for binaries not built
+				// by "go install".
```

Comment:
I got "(devel)" when I did this with go install

Reply:
Fixed in 146feeeaab4f65c64321fa494e0c22d586288ba3.